### PR TITLE
update the prometheus_client_golang dependency to the latest version: v0.9.2

### PR DESCRIPTION
**Pull request by tariq1890**
Using the latest version of client_golang from prometheus.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -139,7 +139,7 @@ func main() {
 	collectorBuilder.WithKubeClient(kubeClient)
 
 	ksmMetricsRegistry := prometheus.NewRegistry()
-	ksmMetricsRegistry.Register(prometheus.NewProcessCollector(os.Getpid(), ""))
+	ksmMetricsRegistry.Register(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
```
Comment (tariq1890) [inline main.go]: As per the documentation, my understanding is that this diff is functionally equivalent.

Comment (andyxning) [reply main.go]: @tariq1890 Yes. It is the recommended usage according to the doc.

**Comment by tariq1890** [comment]
Thanks @andyxning

**Pull request merged**